ipfs-cluster-ctl/main.go

```diff
@@ -398,34 +380,14 @@ although unpinning operations in the cluster may take longer or fail.
 							return nil
 						}
 
-						// handle boolean flags
-						switch {
-						case !c.Bool("no-status"):
-							time.Sleep(1000 * time.Millisecond)
-							resp, cerr := globalClient.Status(ci, false)
-							formatResponse(c, resp, cerr)
-						case c.Bool("wait"):
-							checkFreq := 1 * time.Second
-							waitTimeout := c.Duration("wait-timeout")
-							ctx := context.Background()
-
-							if waitTimeout > checkFreq {
-								var cancel context.CancelFunc
-								ctx, cancel = context.WithTimeout(ctx, waitTimeout)
-								defer cancel()
-							}
-
-							fp := client.StatusFilterParams{
-								Cid:       ci,
-								Local:     false,
-								Target:    api.TrackerStatusUnpinned,
-								CheckFreq: checkFreq,
-							}
-
-							status, cerr := globalClient.WaitFor(ctx, fp)
-							formatResponse(c, status, cerr)
-						}
-
+						waitAndFormatStatusResponse(
+							c,
+							c.Bool("wait"),
+							c.Bool("no-status"),
```

Comment:
It seems weird that `waitAndFormatStatusResponse` would handle the `--no-status` flag. Maybe `handleResponseFormatFlags` would be a better name and then retrieve `wait` and `no-status` from `*cli.Context`. Or the other way, could be to handle `no-status` outside of `waitAndFormatStatusResponse`. Not sure which would be better? @hsanjuan

Reply:
did the first